### DISCUSSION TITLE
Pixi.js mask don't work in one case

### DISCUSSION THREAD

**Discussion by jacobdab** [pixijs/pixijs#8656] [Q&A]
Hey,

I use mask to hide scrolling of items outside of container, but for some reason in one case it stopped working. Can't say what change did make it, cause didn't program it for about half year and after comeback I notice it. What's strange same code works on diffrent views.

```
const accountsContainer = createContainer('accountsContainer', true, true, true); // containerName, making Window object, make it container fullscreen, interactive
    setSpriteXYLocation(accountsContainer, signUpPositions.CHOOSE_ACCOUNT.ACCOUNTS.START);
    let availableAccounts = await getUserAccounts(user._id);
    renderAccountFrame(commonSH, signUpSH, accountsContainer, availableAccounts); // render list of accounts

    const mask = new PIXI.Sprite(PIXI.Texture.WHITE);
    mask.width = accountsContainer.width;
    mask.height = accountsContainer.height;
    accountsContainer.mask = mask;
    accountsContainer.addChild(mask);
```

First video show's how this white texture behaves.

Second video show's same texture attached to container as mask and that she don't hide it where it suppose too.

![2022-09-15_21h49_56](https://user-images.githubusercontent.com/45916043/190808026-6f2564ad-ab6d-4058-b4a7-9433431d0e6d.gif)
![2022-09-15_21h55_47](https://user-images.githubusercontent.com/45916043/190808050-887d025f-6350-4696-946f-bb710e825672.gif)

**Comment by bigtimebuddy** [comment]
Can you provide a standalone reproduction? What version of PixiJS are you using?
